backend/globaleaks/handlers/admin/context.py

```diff
@@ -135,6 +137,7 @@ def db_create_context(session, tid, user_session, request, language):
     :param language: The request language
     :return: The created context
     """
+    logging.debug(user_session)
```

Comment:
SECURITY ISSUE: this log line used probably for debugging logs a user session on the log of the system.